src/toolkit/res_buf.h

```diff
@@ -255,6 +284,8 @@ class ResBuf {
     } else if (rs_present_.count(m) == 1) {
       throw KeyError("duplicate resource push attempted");
     }
+    // strip package id and set as default
+    m->ChangePackageId();
```

Comment:
Don't we want to allow packaging to be preserved in some ResBuf's. Storage should often keep things packaged?  Is this a property of the ResBuf?

Reply:
That's fair, will remove

Reply:
Though we did agree (from #1657 ) "strip package type on Push (becomes implicit type)"

So maybe there should be a `keep_packaging` bool?

Reply:
Or the opposite `bulk_storage` bool?